Sources/MapboxCoreNavigation/NavigationSettings.swift

```diff
@@ -123,12 +125,23 @@ public class NavigationSettings {
      
      If not specified (`nilled`), default value will be used.
      
-     You can override this property by using `NavigationSettings.initialize(directions:tileStoreConfiguration:navigationRouterType:alternativeRouteDetectionStrategy:utilizeSensorData:navigatorPredictionInterval:)` method.
+     You can override this property by using `NavigationSettings.initialize(directions:tileStoreConfiguration:navigationRouterType:alternativeRouteDetectionStrategy:utilizeSensorData:navigatorPredictionInterval:liveIncidentsOptions:)` method.
      */
     public var navigatorPredictionInterval: TimeInterval? {
         state.navigatorPredictionInterval
     }
     
+    /**
+     Configuration on how live incidents on a most probable path are detected.
+     
+     You can override this property by using `NavigationSettings.initialize(directions:tileStoreConfiguration:navigationRouterType:alternativeRouteDetectionStrategy:utilizeSensorData:navigatorPredictionInterval:liveIncidentsOptions:)` method.
+     
+     If set to `nil`, live incidents are turned off (by default).
+     */
+    public var liveIncidentsOptions: IncidentsOptions? {
```

Comment:
If enabled, is this option going to affect billing anyhow? If so, should we mention this?

Reply:
It should not. Billing we are tracking on platform side is about trips counting, which is not affected by this change. Billing feature-wise is done per user token which is outside of NavSDK scope.